File: JuliaLang/ABC001/ABC001_A_v01.jl
# AtCoder Beginner Contest 001 A
# [URL]      https://atcoder.jp/contests/abc001/tasks/abc001_1
# [compiler] Version 0.5.0

#==================================================================================================#

function abc001_A()

	# STEP.01
	# get the values of height of snow
	buf_readline = Base.readline( Base.STDIN, keep=Bool=false )
	H_1, H_2     = Base.split( buf_readline, " " )

	# STEP.02
	# calculate and output the change of the height of snow
	Base.println( Base.STDOUT, H_1 - H_2 )

	return Core.nothing

end

#==================================================================================================#

Main.abc001_A()

File: C/ABC001/ABC001_A_v01.c
// AtCoder Beginner Contest 001 A
// [URL]      https://atcoder.jp/contests/abc001/tasks/abc001_1
// [compiler] C (GCC 5.4.1)
// 
#include <stdio.h>

int main (void)
{
	/* variables for main process */
	int height_1st;
	int height_2nd;

	// STEP.01
	// read the values of the height of snow
	scanf( "%d", &height_1st );
	scanf( "%d", &height_2nd );

	// STEP.02
	// output the change of the height of snow
	printf( "%d\n", height_1st - height_2nd );

	// STEP.TRUE_END
	return 0;
}
